### PR TITLE
Add config flow to `android_ip_webcam` documentation

### PR DESCRIPTION
**Pull request by engrbm87**
## Proposed change

Add config flow to `android_ip_webcam` documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/76222
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/android_ip_webcam.markdown [hunk 1]
```diff
@@ -33,144 +36,14 @@ There is currently support for the following device types within Home Assistant:
...
-      - video_recording
-  - host: 192.168.1.203
-    port: 8000
-    sensors:
-      - light
-    switches:
-      - torch
-```
-
 ## Alternate Configuration Method
```
Comment (MartinHjelmare) [inline source/_integrations/android_ip_webcam.markdown]: Let's remove all of the below too.

**Review by MartinHjelmare — DISMISSED** [review]
Thanks!

File: source/_integrations/android_ip_webcam.markdown [hunk 2]
```diff
@@ -9,13 +9,16 @@ ha_category:
...
   - binary_sensor
   - camera
   - sensor
   - switch
 ha_integration_type: integration
+ha_codeowners:
+  - '@engrbm87'
 ---
 
 The `android_ip_webcam` integration connects with Android IP Webcam to turn any Android phone or tablet into a network camera with multiple viewing options.
```
Comment (frenck) [inline source/_integrations/android_ip_webcam.markdown]:
```suggestion
The Android IP Webcam integration connects with Android IP Webcam to turn any Android phone or tablet into a network camera with multiple viewing options.
```

**Review by frenck — APPROVED** [review]
Thanks, @engrbm87 👍

**Pull request merged**